protocols/v1/Cargo.toml

```diff
@@ -1,11 +1,15 @@
 [package]
 name = "sv1_api"
 version = "1.0.1"
-authors = ["user"]
+authors = ["The Stratum v2 Developers"]
 edition = "2018"
+readme = "README.md"
 description = "API for bridging SV1 miners to SV2 pools"
-license = "MIT OR Apache-2.0"
+documentation = "https://github.com/stratum-mining/stratum"
+license = "MIT + Apache-2.0"
```

Comment:
this is or, not and

Reply:
I didn't get it, Can you elaborate?

Reply:
You changed the license wording from `MIT <OR> Apache` to `MIT <AND> Apache`, it should stay as OR (i.e., don't change OR to +)

Reply:
As its used `+` everywhere except this

Reply:
umm not really.. take a look into the files please. it is currently either `MIT` or `MIT OR Apache` and all of the changes modified it to `MIT + Apache`. Anyway, saying `MIT AND Apache` is confusing as AND implies both licenses forced together, while its either this or that, however the user wants.

Reply:
Actually, 'MIT + Apache' means users can choose either the MIT or Apache 2.0 license. It doesn’t mean both licenses apply together, just that the user has the option to pick one.

Reply:
turns out this is actually a big deal 😛

https://github.com/stratum-mining/stratum/issues/1217